src/components/card/card.component.js

```diff
@@ -57,17 +57,21 @@ const Card = ({
 
   const onClickHandler = interactive ? handleClick : null;
 
+  const interactiveProps = interactive && {
+    tabIndex: 0,
+    type: "button",
+  };
+
```

Comment:
I'm not sure if it matters in this case but ternary is more clear for me
<img width="301" alt="Zrzut ekranu 2021-08-2 o 10 46 49" src="https://user-images.githubusercontent.com/77274590/127832309-54250c36-f77e-4b1c-906b-918ca2699c35.png">

Reply:
Thank you for your suggestion, anyway I would leave it as it is now. We don't need the `undefined` as we don't want to do anything with it. All we need is just `tabIndex` and `type` prop if the card is interactive. I think this is clear enough. We use this approach in our project very often

Reply:
Thank you for your suggestion. If the interactive will be `false` or `undefined` it won't change anything as none of these props will be shown in DOM. Anyway I think that undefined approach is cool. I'll change it